gravitee/values.yaml

```diff
@@ -25,6 +25,19 @@ oauth:
 inMemoryAuth:
   enabled: true
 
+# Default password "admin", use bcrypt to generate a new one
+adminPasswordBcrypt: aaa
+
+smtp:
+  host: smtp.example.com
```

Comment:
Why not also providing the item for `enabled` property ?

Reply:
Fixed